rootfs/api/migrations/0019_auto_20160930_0645.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.1 on 2016-09-30 06:45
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0018_auto_20160908_1748'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='build',
+            name='failed',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AddField(
+            model_name='config',
+            name='failed',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AddField(
```

Comment:
we use it in one place https://github.com/deis/controller/pull/1095/files#diff-5997d406397fa289ab187c6503e4d8b0R85 but i will see if can move away from that which i am sure we can as i have another PR which changes that code but it depends on what we want to do with the changes in the proctypes.
That is the reason i had created this PR https://github.com/deis/controller/issues/1094 to know the expected behaviors.